tests/unit_tests/test_aws.py

```diff
@@ -0,0 +1,23 @@
+from unittest.mock import patch
```

Comment:
Awesome! Thanks for adding the unit test!

Reply:
🚀